seq2science/rules/merge_replicates.smk

```diff
@@ -83,9 +87,9 @@ if "replicate" in samples:
         input:
             unpack(get_merge_replicates),
         output:
-            temp(sorted(expand("{trimmed_dir}/merged/{{replicate}}{{fqext}}_trimmed.{fqsuffix}.gz", **config))),
+            temp(sorted(expand("{trimmed_dir}/{{replicate}}{{fqext}}_trimmed.{fqsuffix}.gz", **config))),
         wildcard_constraints:
-            replicate=any_given("replicate", "control"),
+            replicate=any_given("replicate"),
```

Comment:
what was going on here?

Reply:
Control samples needed to be downloaded and "merged" as well. However if we remove the /merged/ folder, then that is not necessary anymore.